package.json

```diff
@@ -10,6 +10,7 @@
     "packages/*"
   ],
   "scripts": {
+    "analysis": "./bin/run-semgrep.sh",
```

Comment:
Semgrep is just a tool for static analysis. It's better to incapsulate it behind the function name.
btw: in case if will switch the function implementation (change semgrep to sonarQube or etc) script name should be the same